README.md

```diff
@@ -3,7 +3,8 @@
 # Rails Blog App
 
 > The Blog app is Roby on rails website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
-> In this project for the blog app i had to change the validation for the differentmodels as well as writeunit tests for them.
+> In this project for the blog app i had to change the validation for the different models as well as write unit tests for them. Also i had to include forms for the different models.
+
 
```

Comment:
- [ ] Kindly list down all the commands required to run the project on the user's local machine, eg, `bundle install`, `rails db:create`, etc.

Reply:
Hello @elisha2kyakpo1  all changes resuested have been resolved